src/app/pages/contact/form.tsx

```diff
@@ -95,7 +95,9 @@ function useAfterSubmit(setSubmitted?: SetSubmitted) {
 
 function useSubjectWithInitialization() {
     const isEmbedded = useIsEmbedded();
-    const initialValue = isEmbedded ? 'OpenStax Assignable' : 'General';
+    const searchParams = new window.URLSearchParams(useLocation().search);
```

Comment:
This tests to see whether there are multiple body parameters passed in. For the normal embedded form, there will be only one. Assignable passes multiple. Previously, the embedded form was set up to default to Assignable.